webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const path = require('path');
+const webpack = require('webpack');
+
+
+const base = {
+    devtool: 'cheap-module-source-map',
+    module: {
+        rules: [{
+            test: /\.jsx?$/,
```

Comment:
I don't think we'll ever need jsx, will we?

Reply:
The rule checks for `.js` or `.jsx`. We do need .js, so I can change the rule, I just left it as-is for consistency with other scratch repos